<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1240</width>
    <height>980</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MOV MetaEdit</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../Resource/Resource.qrc">
    <normaloff>:/Image/Icon.png</normaloff>:/Image/Icon.png</iconset>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="Central_Widget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QStackedWidget" name="StackedWidget">
      <widget class="QWidget" name="Tech_Page">
        <layout class="QGridLayout" name="Tech_Page_Layout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="1" column="0">
          <widget class="TechTableWidget" name="Tech_Table_Widget">
           <property name="contextMenuPolicy">
            <enum>Qt::CustomContextMenu</enum>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::AllEditTriggers</set>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <property name="sortingEnabled">
            <bool>true</bool>
           </property>
           <property name="columnCount">
            <number>0</number>
           </property>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="Name">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Technical MetaData editor</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
      </widget>
      <widget class="QWidget" name="Adid_Page">
        <layout class="QGridLayout" name="Adid_Page_Layout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="1" column="0">
          <widget class="TableWidget" name="Table_Widget">
           <property name="contextMenuPolicy">
            <enum>Qt::CustomContextMenu</enum>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::AllEditTriggers</set>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <property name="sortingEnabled">
            <bool>true</bool>
           </property>
           <property name="columnCount">
            <number>0</number>
           </property>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="Name">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Universal Ad ID editor</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="Menu_Bar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="Menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="Menu_File_Open_Files"/>
    <addaction name="Menu_File_Open_Directory"/>
    <addaction name="Menu_File_Close"/>
    <addaction name="Menu_File_Close_All"/>
    <addaction name="separator"/>
    <addaction name="Menu_File_Save_All"/>
    <addaction name="separator"/>
    <addaction name="Menu_File_Quit"/>
   </widget>
   <widget class="QMenu" name="Menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="Menu_View_Technical"/>
    <addaction name="Menu_View_AdId"/>
    <addaction name="Menu_View_Tech_Separator"/>
    <addaction name="Menu_View_Tech_Clef"/>
    <addaction name="Menu_View_Tech_Prof"/>
    <addaction name="Menu_View_Tech_Enof"/>
    <addaction name="Menu_View_Tech_Pasp"/>
    <addaction name="Menu_View_Tech_Wscl"/>
    <addaction name="Menu_View_Tech_Fiel"/>
    <addaction name="Menu_View_Tech_Colr"/>
    <addaction name="Menu_View_Tech_Gama"/>
    <addaction name="Menu_View_Tech_Clap"/>
    <addaction name="Menu_View_Tech_Dpri"/>
    <addaction name="Menu_View_Tech_Dlum"/>
    <addaction name="Menu_View_Tech_Mcll"/>
    <addaction name="Menu_View_Tech_Mfal"/>
    <addaction name="Menu_View_Tech_Chan"/>
   </widget>
   <widget class="QMenu" name="Menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="Menu_Help_Help"/>
    <addaction name="Menu_Help_About"/>
   </widget>
   <addaction name="Menu_File"/>
   <addaction name="Menu_View"/>
   <addaction name="Menu_Help"/>
   <addaction name="Menu_New_Version"/>
  </widget>
  <widget class="QToolBar" name="Tool_Bar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="Status_Bar"/>
  <action name="Menu_File_Quit">
   <property name="text">
    <string>Q&amp;uit</string>
   </property>
   <property name="toolTip">
    <string>Quit the application</string>
   </property>
   <property name="statusTip">
    <string>Quit the application</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
  <action name="Menu_File_Open_Files">
   <property name="icon">
    <iconset resource="../../Resource/Resource.qrc">
     <normaloff>:/Image/File_Open_Files.png</normaloff>:/Image/File_Open_Files.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open file(s)...</string>
   </property>
   <property name="toolTip">
    <string>Open file(s)</string>
   </property>
   <property name="statusTip">
    <string>Open file(s)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="Menu_Help_About">
   <property name="icon">
    <iconset resource="../../Resource/Resource.qrc">
     <normaloff>:/Image/About.png</normaloff>:/Image/About.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About...</string>
   </property>
   <property name="toolTip">
    <string>About MOV MetaEdit</string>
   </property>
   <property name="statusTip">
    <string>About MOV MetaEdit</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="Menu_File_Open_Directory">
   <property name="icon">
    <iconset resource="../../Resource/Resource.qrc">
     <normaloff>:/Image/File_Open_Directory.png</normaloff>:/Image/File_Open_Directory.png</iconset>
   </property>
   <property name="text">
    <string>Open directory...</string>
   </property>
   <property name="statusTip">
    <string>Open directory</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="Menu_File_Close">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../../Resource/Resource.qrc">
     <normaloff>:/Image/File_Close.png</normaloff>:/Image/File_Close.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Close</string>
   </property>
   <property name="toolTip">
    <string>Close selected file(s)</string>
   </property>
   <property name="statusTip">
    <string>Close selected file(s)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="Menu_File_Close_All">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Close &amp;All</string>
   </property>
   <property name="toolTip">
    <string>Close all opened files</string>
   </property>
   <property name="statusTip">
    <string>Close all opened files</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="Menu_File_Save_All">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../../Resource/Resource.qrc">
     <normaloff>:/Image/File_Save.png</normaloff>:/Image/File_Save.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="toolTip">
    <string>Save all modified files</string>
   </property>
   <property name="statusTip">
    <string>Save all modified files</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="Menu_Help_Help">
   <property name="icon">
    <iconset resource="../../Resource/Resource.qrc">
     <normaloff>:/Image/Help.png</normaloff>:/Image/Help.png</iconset>
   </property>
   <property name="text">
    <string>H&amp;elp...</string>
   </property>
   <property name="toolTip">
    <string>Documentation</string>
   </property>
   <property name="statusTip">
    <string>Documentation</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="Menu_New_Version">
   <property name="text">
    <string>New version released</string>
   </property>
   <property name="statusTip">
    <string>MOVMetaEdit version %1 is available</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="Menu_View_Tech_Separator">
   <property name="separator">
    <bool>true</bool>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="Menu_View_Tech_Clef">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Clean Ap.</string>
   </property>
   <property name="toolTip">
    <string>Show clean aperture column</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="Menu_View_Tech_Prof">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Prod Ap.</string>
   </property>
   <property name="toolTip">
    <string>Show production aperture column</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="Menu_View_Tech_Enof">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Enc. Ap.</string>
   </property>
   <property name="toolTip">
    <string>Show encoded pixels dimension column</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="Menu_View_Tech_Pasp">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>PAR</string>
   </property>
   <property name="toolTip">
    <string>Show PAR column</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="Menu_View_Tech_Wscl">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>w-scale</string>
   </property>
   <property name="toolTip">
    <string>Show width scale column</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="Menu_View_Tech_Fiel">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Field</string>
   </property>
   <property name="toolTip">
    <string>Show field handling column</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="Menu_View_Tech_Colr">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Color</string>
   </property>
   <property name="toolTip">
    <string>Show color column</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="Menu_View_Tech_Gama">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Gamma</string>
   </property>
   <property name="toolTip">
    <string>Show gamma column</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="Menu_View_Tech_Clap">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Aperture</string>
   </property>
   <property name="toolTip">
    <string>Show clean aperture column</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="Menu_View_Tech_Dpri">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Display Primaries</string>
   </property>
   <property name="toolTip">
    <string>Show display primaries column</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="Menu_View_Tech_Dlum">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Luminance</string>
   </property>
   <property name="toolTip">
    <string>Show luminance column</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="Menu_View_Tech_Mcll">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Max. CLL</string>
   </property>
   <property name="toolTip">
    <string>Show maximum content light level column</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="Menu_View_Tech_Mfal">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Max. FALL</string>
   </property>
   <property name="toolTip">
    <string>Show maximum frame average light level column</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="Menu_View_Tech_Chan">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Chan</string>
   </property>
   <property name="toolTip">
    <string>Show audio channels description labels column</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <actiongroup name="Menu_View_ActionGroup">
  <action name="Menu_View_Technical">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../Resource/Resource.qrc">
     <normaloff>:/Image/View_Technical.png</normaloff>:/Image/View_Technical.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Technical</string>
   </property>
   <property name="toolTip">
    <string>Technical metadata editor</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="Menu_View_AdId">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../Resource/Resource.qrc">
     <normaloff>:/Image/View_AdId.png</normaloff>:/Image/View_AdId.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Ad-Id</string>
   </property>
   <property name="toolTip">
    <string>Ad-Id metadata editor</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  </actiongroup>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../Resource/Resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>Menu_File_Quit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
